src/tiled/scriptfileinfo.cpp

```diff
@@ -242,3 +242,74 @@ QString ScriptFileInfo::symLinkTarget(const QString &file) const
 {
     return QFileInfo(file).symLinkTarget();
 }
+
+QString ScriptFileInfo::cleanPath(const QString &file) const
+{
+    return QDir().cleanPath(file);
```

Comment:
This is a static method, so it doesn't need an instance. It should be `return QDir::cleanPath(file)`.

Reply:
sounds good.